WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
I updated and integrated old code from the VIET mod exclusively for WtWSMS. Some of which that has never been released before, which was nonetheless brought in for flavour.